new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/LoggingExporter.java

```diff
@@ -0,0 +1,474 @@
+        String.format("Create Elasticsearch deployment failed with ApiException for %s in namespace %s",
+            elasticsearchName, namespace));
+    logger.info("Check if Elasticsearch deployment {0} is ready in namespace {1}",
+        elasticsearchName, namespace);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for Elasticsearch deployment {0}"
+                + "to be completed in {1} namespace (elapsed time {2}ms, remaining time {3}ms)",
+                elasticsearchName,
+                namespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(Deployment.isReady(elasticsearchName, labels, namespace));
+
+    // create Elasticsearch service CR
+    logger.info("Create Elasticsearch service CR for {0} in namespace {1}",
+        elasticsearchName, namespace);
+    V1Service elasticsearchService = createElasticsearchServiceCr(params);
+
+    // create Elasticsearch service
+    logger.info("Create Elasticsearch service {0} in namespace {1}", elasticsearchName, namespace);
+    boolean serviceCreated = assertDoesNotThrow(() -> Kubernetes.createService(elasticsearchService),
+        String.format("Create Elasticsearch service failed with ApiException for %s in namespace %s",
+            elasticsearchName, namespace));
+    assertTrue(serviceCreated, String.format(
+        "Create Elasticsearch service failed with ApiException for %s in namespace %s",
+            elasticsearchName, namespace));
+
+    // check that Elasticsearch service exists in its namespace
+    logger.info("Check that Elasticsearch service {0} exists in namespace {1}",
+        elasticsearchName, namespace);
+    checkServiceExists(elasticsearchName, namespace);
+
+    return true;
+  }
+
+  /**
+   * Install Kibana.
+   *
+   * @param params the parameters to install Kibana
+   * @return true if the Kibana is successfully installed, false otherwise.
+   */
+  public static boolean installKibana(LoggingExporterParams params) {
+    String kibanaName = params.getKibanaName();
+    String namespace = params.getLoggingExporterNamespace();
+    Map labels = new HashMap<String, String>();
+    labels.put("app", kibanaName);
+
+    // create Kibana deployment CR
+    logger.info("Create Kibana deployment CR for {0} in namespace {1}", kibanaName, namespace);
+    V1Deployment kibanaDeployment = createKibanaDeploymentCr(params);
+
+    // create Kibana deployment
+    logger.info("Create Kibana deployment {0} in namespace {1}", kibanaName, namespace);
+    boolean deploymentCreated = assertDoesNotThrow(() -> Kubernetes.createDeployment(kibanaDeployment),
+        String.format("Create Kibana deployment failed with ApiException for %s in namespace $s",
+            kibanaName, namespace));
+    assertTrue(deploymentCreated,
+        String.format("Create Kibana deployment failed with ApiException for %s in %s namespace",
```

Comment:
change the message as if it failed with ApiException it is captured in the above line

Reply:
changed